### DISCUSSION TITLE
Why does it keeps trying to access Searx when no agent even has the Search Engine command enabled?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [Josh-XT/AGiXT#800] [Q&A]
searx.space to be more specific

**Comment by Josh-XT** [comment]
Smart functions search the web for information relevant to the user input prior to trying to respond.  If you select to use a smart function, it will always search rather the agent has a command for that enabled or not.

Please take a look at the core concepts. https://josh-xt.github.io/AGiXT/2-Concepts/0-Core%20Concepts.html

**Reply by TiagoTiago** [reply]
It seems to keep doing it even if I delete all agents, so no chance I pressed the "smart" checkbox in the Interact tab.
